dev-ruby/gh/gh-0.14.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+RUBY_FAKEGEM_RECIPE_TEST="none"
+
+inherit ruby-fakegem
+
+DESCRIPTION="multi-layer client for the github api v3"
+HOMEPAGE="http://gh.rkh.im/"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+ruby_add_bdepend "
+	dev-ruby/rspec
+	dev-ruby/webmock
+"
+
+ruby_add_rdepend "
```

Comment:
Some of these dependencies are versioned, that also needs to be mimicked here. rubygems will check the gemspec versions and won't load if things don't match up. See e.g. https://rubygems.org/gems/gh